_data/webvitals.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2020 Google Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+const metadata = require("./metadata.json");
+
+module.exports = function () {
+  // Check opt-in to send CWV metrics to GA.
+  let sendWebVitals = metadata.sendWebVitals;
+
+  // Now check that web-vitals.js exists.
+  let cwvLibraryExists = false;
+
+  if (sendWebVitals) {
+    try {
+      require("../js/web-vitals.js").getCLS;
```

Comment:
Why is this not always present?

Reply:
I was thinking about a case where the user opts-out of sending metrics (`sendWebVitals === false`), then do not bother checking for the library. 

I could instead check `sendWebVitals` and `cwvLibraryExists` independently? Open to your thoughts on a more robust way to accomplish these checks!

Reply:
Yeah, I think I'd prefer this to fail regularly if the user opted into sending CWV via the json file.